linode/objbucket/tmpl/template.go

```diff
@@ -123,15 +123,15 @@ func ForceDelete_Empty(t *testing.T) string {
 	return acceptance.ExecuteTemplate(t, "object_bucket_force_delete_empty", nil)
```

Comment:
Error: linode/objbucket/resource_test.go:639:37: cannot use t (variable of type *acceptance.WrappedT) as *"testing".T value in argument to tmpl.ForceDelete_Empty

Should this be `t testing.TB` as well?

Reply:
Nice find! Somehow I missed this lol